Prepare release 1.6.0 (#179)
Esri/pyprt: CHANGELOG.md

* Internal update to PRT 2.7.8538 (CityEngine 2022.1)
* Rebuilt test RPKs with CityEngine 2022.1
* `pyprt_arcgis` module: added handling of polygons holes (inner rings) based on Shapely

> * Modified PRT and PyBind11 fetch in the build system

Reviewer: let's remove this item and expand the one in "added"

* Simplified the environment files to the minimal necessary packages
* Updated environment files

> * Removed unnecessary PRT import libraries

Reviewer: ```suggestion
* Stop shipping unnecessary MSVC import libraries for PRT in the packages.
```

* Build Python with SSL support in Linux Docker images
* Improved conda environments pipeline

### Added
* Added pre-built wheels and conda packages for Python 3.10

> * New setup.py flag for externally provided PyBind11 sources

Reviewer: ```suggestion
* New CMake flag `pybind11_DIR` and env var `PYBIND11_DIR` for setup.py to let users specify a custom copy of PyBind11.
```

> * New example 10 script for automatic update of Web Scene 3D visualization

Reviewer: ```suggestion
* [New example (number 10)](https://github.com/esri/pyprt-examples) about updating Scene Layers used in Web Scenes.

```
